docs/GraphNode.fbmd

```diff
@@ -270,111 +270,116 @@ Returns the `perms` property for the page as an `array` if present. (Only availa
 <card>
 ## GraphAlbum Instance Methods {#album-instance-methods}
 
-The `GraphAlbum` collection represents an [Album](https://developers.facebook.com/docs/graph-api/reference/album) Graph node.
```

Comment:
I add this line to my generator, and I also update `GraphEvent`, `GraphLocation`, `GraphPicture`, `GraphAchievement` collections.